dashboard/pom.xml

```diff
@@ -188,22 +188,22 @@
                     <plugin>
                         <artifactId>maven-antrun-plugin</artifactId>
                         <executions>
-                          <execution>
-                            <id>install-yarn</id>
-                            <phase>compile</phase>
-                            <goals>
-                              <goal>run</goal>
-                            </goals>
-                            <configuration>
-                              <target>
-                                <!-- install yarn -->
-                                <exec dir="${basedir}" executable="npm" failonerror="true">
-                                  <arg value="install" />
-                                  <arg value="yarn" />
-                                </exec>
-                              </target>
-                            </configuration>
-                          </execution>
+                            <execution>
```

Comment:
are you sure that this change is related to this pr?

Reply:
So, this change has no direct link with `Adding tracking of STARTING/STOPPING recovered k8s/OS runtimes` BUT it is a result of execution `'mvn sortpom:sort license:format fmt:format'` in root of Che Project. So, `pom.xml` in dashboard was commited into master with broken format. I would merge this change in this PR not to waste time on creating a new PR just for fixing format in one file.